templates/summit/shared/_latest-blogs_summit.html

```diff
@@ -0,0 +1,46 @@
+<section class="p-section--deep">
```

Comment:
Instead of making new templates every time we want to display a blog section, I think we should be extending the [re-usable jinja template](https://github.com/canonical/ubuntu.com/blob/main/templates/shared/_latest_news_strip.html) to fit the use case.
Having all the requirements will also help in the future when we come to making an actual jinja template

Reply:
Yes you are absolutely right, there are smaller things that wouldn't fit in the above template(for instance difference in headings h2 in design vs h3 in re-usable template) and therefore I had to make new one.
I think we need to make a more configurable template for this, may be a macro?